includes/meta.php

```diff
@@ -160,9 +160,26 @@ function add_meta_tags() {
 <meta name="google-site-verification" content="<?php echo htmlentities( $gw_verify ); ?>">
 <?php endif; ?>
 
-<?php // Preload Font Awesome ?>
-<link rel="preload" href="<?php echo THEME_FONT_URL; ?>/font-awesome/fontawesome-webfont.woff2?v=<?php echo THEME_FA_VERSION; ?>" as="font" type="font/woff2" crossorigin>
-
+<?php // Preload Font Awesome
+$fonts_array = array(
+	`/font-awesome-pro/fa-brands-400.woff2`,
+	`/font-awesome-pro/fa-duotone-900.woff2`,
+	`/font-awesome-pro/fa-light-300.woff2`,
+	`/font-awesome-pro/fa-regular-400.woff2`,
+	`/font-awesome-pro/fa-sharp-light-300.woff2`,
+	`/font-awesome-pro/fa-sharp-regular-400.woff2`,
+	`/font-awesome-pro/fa-sharp-solid-900.woff2`,
+	`/font-awesome-pro/fa-sharp-thin-100.woff2`,
+	`/font-awesome-pro/fa-solid-900.woff2`,
+	`/font-awesome-pro/fa-thin-100.woff2`,
+	`/font-awesome-pro/fa-v4compatibility.woff2`,
```

Comment:
[Backticks mean something very particular in PHP](https://www.php.net/manual/en/language.operators.execution.php) - we need to update these to single quotes, please.

Reply:
I'll make the changes.